zerver/webhooks/trello/doc.md

```diff
@@ -34,12 +36,17 @@ Get Zulip notifications from your Trello boards!
     macOS or Linux, you very likely already do. If you're running
     Windows you may or may not.  If you don't have Python, follow the
     installation instructions
-    [here](https://realpython.com/installing-python/). Note that you
-    do not need the latest version of Python; anything 2.7 or higher
-    will do.
+    [here](https://realpython.com/installing-python/).
+
+    !!! tip ""
```

Comment:
I was wondering whether we were using tips and warnings a bit excessively in this doc, but the buddy reviewers reported that it looked fine.

Reply:
I think it looks alright too.